.travis.yml

```diff
@@ -1,25 +1,37 @@
-language: c
+language: cpp
```

Comment:
ok thanks for your feedback @malaterre .

@rouault I think we can proceed and merge now.